test/smoke/src/main.ts

```diff
@@ -290,7 +290,8 @@ async function setup(): Promise<void> {
 	console.log('*** Test data:', testDataPath);
 	console.log('*** Preparing smoketest setup...');
 
-	await ensureStableCode();
+	// {{SQL CARBON EDIT}} Smoketests no longer need to download VS Code since they run against ADS
```

Comment:
Edit comments should go on the same line as the thing being commented out whenever possible